packages/libqtxdg/build.sh

```diff
@@ -1,11 +1,11 @@
 TERMUX_PKG_HOMEPAGE=https://lxqt.github.io
 TERMUX_PKG_DESCRIPTION="Qt 5 implementation of freedesktop.org XDG specifications"
-TERMUX_PKG_LICENSE="LGPL-2.1"
+TERMUX_PKG_LICENSE="LGPL-2.0"
```

Comment:
The license should be <a href="https://github.com/lxqt/libqtxdg/blob/master/COPYING">LGPL-2.1</a>. No need to change this.